pkg/services/sqlstore/migrations/ualert/permissions.go

```diff
@@ -0,0 +1,162 @@
+package ualert
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/util"
+
+	"github.com/grafana/grafana/pkg/infra/metrics"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+// getOrCreateGeneralFolder returns the general folder under the specific organisation
+// If the general folder does not exist it creates it.
+func (m *migration) getOrCreateGeneralFolder(orgID int64) (*models.Dashboard, error) {
```

Comment:
Any reason for creating the general folder? I do not think we have ever created this folder, but might be a good idea still as you cannot set permissions on general today as it does not actually exist

Reply:
I think the main reason we said was to lessen the chance that we would pollute the root with too many items during the migration.